flask_sillywalk/tests/test_sillywalk.py

```diff
@@ -0,0 +1,253 @@
+        data = json.loads(s(ret.data))
+        self.assertEqual(data['swaggerVersion'], '1.3')
+        self.assertEqual(data['basePath'], 'http://localhost:5000/api/v1')
+        self.assertEqual(sorted([x["path"] for x in data['apis']]),
+                         ['/cheese.{format}'])
+        self.assertEqual(data['apiVersion'], '1.0')
+        self.assertEqual(data['models'], {})
+        # test cheese
+        ret = app.test_client().get("/api/v1/cheese.json")
+        data = json.loads(s(ret.data))
+        self.assertEqual(data["resourcePath"], "cheese")
+
+    def test_register_blueprint(self):
+        app, registry = self._create_app()
+        bp = self._create_blueprint(app)
+        self._create_register(registry.register, app, bp)
+        ret = app.test_client().get("/api/v1/resources.json")
+        data = json.loads(s(ret.data))
+        self.assertEqual(data['swaggerVersion'], '1.3')
+        self.assertEqual(data['basePath'], 'http://localhost:5000/api/v1')
+        self.assertEqual(sorted([x["path"] for x in data['apis']]),
+                         ['/cheese.{format}', '/holyHandGrenade.{format}'])
+        self.assertEqual(data['apiVersion'], '1.0')
+        self.assertEqual(data['models'], {})
+        # test cheese
+        ret = app.test_client().get("/api/v1/cheese.json")
+        data = json.loads(s(ret.data))
+        self.assertEqual(data["resourcePath"], "cheese")
+        # FIXME this is broken urls shouldn't be added multiple times
+        self.assertEqual([x.endpoint for x in app.url_map.iter_rules()],
+                         ['holyHandGrenade',
+                          'resources',
+                          'cheese',
+                          'blueish.get_a_holy_hand_grenade',
+                          'blueish.toss_the_grenade',
+                          'blueish.get_cheese',
+                          'static'])
+
+    def test_add_register_blueprint(self):
+        app, registry = self._create_app()
+        bp = self._create_blueprint(app)
+        self._create_add_register(registry.add_register, app, bp)
+        ret = app.test_client().get("/api/v1/resources.json")
+        data = json.loads(s(ret.data))
+        self.assertEqual(data['swaggerVersion'], '1.3')
+        self.assertEqual(data['basePath'], 'http://localhost:5000/api/v1')
+        self.assertEqual(sorted([x["path"] for x in data['apis']]),
+                         ['/cheese.{format}'])
+        self.assertEqual(data['apiVersion'], '1.0')
+        self.assertEqual(data['models'], {})
+        # test cheese
+        ret = app.test_client().get("/api/v1/cheese.json")
+        data = json.loads(s(ret.data))
+        self.assertEqual(data["resourcePath"], "cheese")
+        self.assertEqual([x.endpoint for x in app.url_map.iter_rules()],
+                         ['resources',
+                          'cheese',
+                          'blueish.get_cheese',
+                          'static'])
```

Comment:
There is now nearly 100% coverage so it should be easier to add new features without breaking something.